workflow/controller/dag_test.go

```diff
@@ -2861,3 +2861,127 @@ func TestFailsWithParamDAG(t *testing.T) {
 	woc.operate()
 	assert.Equal(t, wfv1.NodeFailed, woc.wf.Status.Phase)
 }
+
+var testLeafContinueOn = `
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  name: build-wf-kpxvm
+spec:
+  arguments: {}
```

Comment:
minor - please clean up this example - all the additional lines make it hard to understand which ones actually influence the test

Reply:
Test is already minimized as per https://github.com/argoproj/argo/pull/4455#discussion_r517486368

Reply:
this is minor - but cant' you remove `arguments: {}` for example?